test/core/util/tls_utils.cc

```diff
@@ -33,7 +33,13 @@ namespace grpc_core {
 
 namespace testing {
 
-TmpFile::TmpFile(absl::string_view data) {
+TmpFile::TmpFile(absl::string_view data) : TmpFile(data, "test") {
+  name_ = CreateTmpFileAndWriteData(data);
```

Comment:
I don't think you want to call this here, since it's already being done in the other ctor that you're delegating to.

Reply:
Good catch